packages/platforms/accton/x86-64/as9716_32d/modules/builds/x86-64-accton-as9716-32d-ioport.c

```diff
@@ -0,0 +1,180 @@
+/*
+ * A hwmon driver for the Accton as9926 24d fan
+ *
+ * Copyright (C) 2016 Accton Technology Corporation.
```

Comment:
Is this copied from somewhere?

Reply:
Modified as as9716 32d.

Reply:
1. Modify as9926 24d as as9716 32d
2. Tested on latest version of ONL 4.14.151-OpenNetworkLinux, kernel 5.x also works.

Reply:
Please mention that in the commit message.

Reply:
The copyright also needs updating, doesn’t it? Maybe use SPDX headers for the licence header too.